pkg/bootstrap/platform/gcp_test.go

```diff
@@ -130,6 +131,7 @@ func TestGCPMetadata(t *testing.T) {
 			for e, v := range tt.env {
 				os.Setenv(e, v)
 			}
+			once = sync.Once{}
```

Comment:
Not used?

Reply:
I need to reset before each test, otherwise the last test in TestGCPMetadata would fail because parseGCPMetadata() is only called once during the first test.